articles/install-fleet-maintained-apps-on-macos-hosts.md

```diff
@@ -44,19 +45,19 @@ The uninstallation process is also visible in the  **Activities** section on the
 
 ## Update the app
 
-Currently, Fleet does not automatically update apps. To update the app, remove the app and re-add it from the Fleet-maintained list on the **Software** page, then reinstall it.
+Currently, Fleet does not automatically update apps. To update the app, remove the app and re-add it from the Fleet-maintained list on the **Software** page, then re-add it.
```

Comment:
Hey! Is this on purpose? Looks like we're saying re-add it twice?

Reply:
It is not. Reverting that as I misread what the instructions were.